src/app/common/directives/oc-file-upload/templates/oc-file-upload.html

```diff
@@ -15,6 +15,7 @@
             ng-click="openModal()" 
             ng-if="!fileUploadModel[fileUploadOptions.keyname][fileUploadOptions.srcKeyname]" 
             value="{{fileUploadOptions.uploadText}}">
+            <i class="fa fa-plus-circle"></i>
```

Comment:
@robertsoniv do you want this removed too? Thought it should be here since the other template has this on the upload button